dll/win32/shell32/CCopyMoveToMenu.h

```diff
@@ -109,3 +109,35 @@ class CMoveToMenu
     UINT GetFileOp() const override { return FO_MOVE; }
     LPCSTR GetVerb() const override { return "moveto"; }
 };
+
+class CCopyAsPathMenu 
+    : public CComCoClass<CCopyAsPathMenu, &CLSID_CopyAsPathMenu>
+    , public CComObjectRootEx<CComMultiThreadModelNoCS>
+    , public IDropTarget
+{
+public:
+    DECLARE_REGISTRY_RESOURCEID(IDR_COPYASPATHMENU)
+    DECLARE_NOT_AGGREGATABLE(CCopyAsPathMenu)
+    DECLARE_PROTECT_FINAL_CONSTRUCT()
+
+    BEGIN_COM_MAP(CCopyAsPathMenu)
+        COM_INTERFACE_ENTRY_IID(IID_IDropTarget, IDropTarget)
+    END_COM_MAP()
+
+    // IDropTarget
+    STDMETHODIMP DragEnter(IDataObject *pdto, DWORD grfKeyState, POINTL ptl, DWORD *pdwEffect) override
```

Comment:
iirc the override keyword is not compatible to older toolchains. It is no hard requirement to change your code, as you are not the first one tainting the ros sources with that keyword. **But I don't want you to uncollapse this comment.** This could never be ported back therefore.

Reply:
I just followed the existing style.

Why did you not say anything when you approved [this PR](https://github.com/reactos/reactos/pull/5873) 5 months ago?!

Reply:
Because I give my approval when that PR consisted only of the single commit:
https://github.com/reactos/reactos/pull/5873/commits/526d42168018fc57338043994ab498f78733a78c
Everything that came afterwards had nothing to do with my approval. And my approval has not been reset by Katayama when he pushed more things on top of that. I will and can also not port back all that stuff of Katayama in its current state, for the same reason. Correct.

Like I said: I didn't even request you to change your code here. I just requested you to not collapse this discussion. You are free to act on or ignore this specific review-comment of mine.